airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/JdbcBufferedConsumerFactory.kt

```diff
@@ -99,12 +99,13 @@ object JdbcBufferedConsumerFactory {
             )
         }
         return parsedCatalog.streams
-            .map { parsedStreamToWriteConfig(namingResolver).apply(it) }
+            .map { parsedStreamToWriteConfig(namingResolver, "").apply(it) }
```

Comment:
nit: named param for the string literal

Reply:
I think embedding the streamConfig into the WriteConfig might make sense. Seems a bit heavy handed for this refactor though

Reply:
(I think you replied to the wrong comment thread, but that makes sense)